IFC4x3/Properties/t/TAMTubesSpacing_3crl3aiib4G91_PCrBXmy0/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TAMTubesSpacing_3crl3aiib4G91_PCrBXmy0" Name="TAMTubesSpacing" UniqueId="e6d6f0e4-b2c9-4440-907e-64cd4b870f00" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Curiosity, what does TAM mean? And should it not be explained in the description?

Reply:
The explanation is already in the description (at least now, don't know if it was added later). Howerver, the name is slightly wrong, it should be "Tubes à manchette".